### PR TITLE
Added YPHIX 50208693 with vendor 'Sunricher' and zigbeeModel 'ZGRC-KEY-009'

### PR DESCRIPTION
**Pull request by mackshot**
see https://zigbee.blakadder.com/Yphix_50208693.html for more details.

I considered to add a new yphix.js, but since vendor is sunricher I decided to add it to sunricher.js

### CODE REVIEW

File: devices/sunricher.js [hunk 1]
```diff
@@ -25,6 +25,18 @@ module.exports = [
...
+        zigbeeModel: ['ZGRC-KEY-009'],
+        model: 'YPHIX 50208693',
+        vendor: 'Sunricher',
+        description: 'Zigbee Wall Remote Control for RGBW, 1 Zone with 2 Scenes',
+        fromZigbee: [fz.command_on, fz.command_off, fz.command_move, fz.command_stop, fz.battery,
+            fz.command_recall, fz.command_step, fz.command_move_to_color, fz.command_move_to_color_temp],
+        toZigbee: [],
+        exposes: [e.battery(), e.action(['on', 'off',
+            'brightness_move_up', 'brightness_move_down', 'brightness_move_stop', 'brightness_step_up', 'brightness_step_down',
+            'recall_*'])],
```
Comment (Koenkk) [inline devices/sunricher.js:38]: Instead of using the wildcard `recall_*` here, please add all the send value (e.g. `recall_1`, etc..)

Comment (mackshot) [reply devices/sunricher.js]: Done :)

**Comment by mackshot** [comment]
You were faster adding the space. Thank you.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**